search/src/test/scala/weco/api/search/services/AggregationsTest.scala

```diff
@@ -148,35 +148,19 @@ class AggregationsTest
             List(WorkAggregationRequest.Format, WorkAggregationRequest.Subject),
           filters = List(
             FormatFilter(List("a")),
-            SubjectLabelFilter(Seq("pGkJTZWwn4"))
+            SubjectLabelFilter(Seq("9SceRNaTEl"))
           )
         )
         whenReady(aggregationQuery(index, searchOptions)) { aggs =>
           val buckets = aggs.format.get.buckets
-          buckets.length shouldBe works.length
+          buckets.length shouldBe 7
```

Comment:
There was no subject in the existing dataset that was present in all works.  However, this is an improvement in that it shows the filter being applied.